index.html

```diff
@@ -46,8 +46,169 @@ <h2>I'm a Software Developer</h2>
             </div>
             </div>
         </div>
-        <div class="shapels">
-            <img src="imakges/Illustration (1).png" alt="">
+    </section>
+    <section>
+        <div class="work-section">
+            <div class="recent-work">
+                <h1>My Recent Work</h1>
```

Comment:
[OPTIONAL]
- [ ] Great job! :rocket: On using the `hx` tag for this title, but I suggest using the `h1` only once in your project, preferably your name. This will help in making your project more semantic.